graylog2-server/src/main/java/org/graylog2/indexer/ranges/MongoIndexRangeService.java

```diff
@@ -79,25 +81,25 @@ public MongoIndexRangeService(MongoConnection mongoConnection,
         this.auditEventSender = auditEventSender;
         this.nodeId = nodeId;
```

Comment:
Don't we need a separate instance for each node?

Reply:
Each node should have it's own vm and within this vm, nodeid will be unique, shouldn't it?